### PR TITLE
feat: add arbiter ipfsconn component

### PR DESCRIPTION
**Pull request by lanzafame**
This is a first pass at solving issue [#1430](https://github.com/ipfs/ipfs-cluster/issues/1430). @hsanjuan I have only done basic tests, `numpin` is incompatible with this dummy ipfshttp connector. I have left the config as is for now but a lot of it is not needed now, i.e. timeouts, disableunpin.

### CODE REVIEW

**Comment by hsanjuan** [comment]
This should be a very simple mock of the ipfsconnector interface. Doesn't need config, most code can go away. Probably faster to write from scratch than to copy/paste+remove.

File: ipfscluster_test.go [hunk 1]
```diff
@@ -59,6 +60,7 @@ var (
 
 	consensus = "crdt"
 	datastore = "badger"
+	connector = "arbiter"
```
Comment (lanzafame) [inline ipfscluster_test.go:63]: @hsanjuan I will change this back to the normal ipfshttp connector, once you are happy with the PR.

**Comment by hsanjuan** [comment]
I am not sure I understand this PR. The idea is to have a peer that does not really pin or anything, using a mock ipfsconn that does nothing. Why is it proxying things to the actual connector?

**Comment by lanzafame** [comment]
@hsanjuan because other parts of cluster still rely on the connector interface? To clarify, are just suggesting a complete empty ipfsconnector interface implementation?

**Pull request closed without merging**